superset-frontend/src/dashboard/components/nativeFilters/FilterConfigModal.tsx

```diff
@@ -82,6 +82,7 @@ const FilterTabs = styled(LineEditableTabs)`
 const FilterTabTitle = styled.span`
   transition: color ${({ theme }) => theme.transitionTiming}s;
   width: 100%;
+  max-width: 200px;
```

Comment:
Can we make this based on a constant, or on grid units?

Reply:
Yes, you are right - it's fixed now. For this amount I would go with constant.